src/main/java/de/hysky/skyblocker/skyblock/fancybars/FancyStatusBars.java

```diff
@@ -372,15 +376,24 @@ public static boolean render(GuiGraphics context, Minecraft client) {
 			statusBar.renderText(context);
 		}
 
-		StatusBarTracker.Resource health = StatusBarTracker.getHealth();
-		statusBars.get(StatusBarType.HEALTH).updateWithResource(health);
+		if (Utils.isInTheRift()) {
+			statusBars.get(StatusBarType.HEALTH).updateValues(player.getHealth() / player.getMaxHealth() / 2, 0, (int) player.getHealth() / 2, player.getMaxHealth(), null);
```

Comment:
Health isn't correct, the bar appears half full despite being at full health. After taking damage and healing back up, it shows health-1 (int issue, I guess)